web/app/themes/mitlib-parent/functions.php

```diff
@@ -169,8 +169,6 @@ function setup_scripts_styles() {
 	wp_register_script( 'menu-toggle', get_template_directory_uri() . '/js/menu.toggle.js', array(), $theme_version, true );
 	wp_register_script( 'parent-production', get_template_directory_uri() . '/js/alerts.js', array( 'dev', 'fonts', 'jquery', 'libchat', 'menu-toggle' ), $theme_version, true );
 
-	wp_register_script( 'parent-interior', get_template_directory_uri() . '/js/core.js', array( 'ga-links' ), $theme_version, true );
```

Comment:
Sorry, this goes too far the other way - we still need the core.js file to be registered (and later enqueued). That file handles things like the expandable panels that get loaded around the site.

The only change needed was to replace `array( 'ga-links' )` with `array()` because now core.js has no dependencies.